docs/src/IconButton.doc.js

```diff
@@ -1,419 +1,862 @@
+    <Image
+        alt="Tropic greens: The taste of Petrol and Porcelain | Interior design, Vintage Sets and Unique Pieces agave"
+        naturalHeight={1}
+        naturalWidth={1}
+        src="https://i.ibb.co/x65Wctf/image.jpg"
+      >
+        <Box height="100%" padding={3} display="flex" justifyContent="end" alignItems="end">
+          <IconButton
+            accessibilityLabel="Share pin"
+            icon="share"
+            size="md"
+            iconColor="white"
+          />
+        </Box>
+      </Image>
+  </Mask>
+</Box>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Selected state"
-    id="selected"
-    defaultCode={`
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="ARIA attributes"
+      description={`
+IconButton conveys the component behavior using iconography. IconButton requires \`accessibilityLabel\`, a text description for screen readers to announce and communicate the represented [Icon](/Icon). In the example below, the screen reader reads: "More Options."
+
+If IconButton is used as a control button to show/hide a Popover-based component, we recommend passing the following ARIA attributes to assist screen readers:
+
+- \`accessibilityControls\`: informs the screen reader that IconButton controls the display of an anchored Popover-based component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityHaspopup\`: informs the screen reader that there’s a Popover-based component attached to IconButton. It populates [aria-haspopup](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+- \`accessibilityExpanded\`: informs the screen reader whether an anchored Popover-based component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 function Example() {
-  const [selected, setSelected] = React.useState(false);
+  const [open, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState([]);
+  const anchorRef = React.useRef(null);
+  const onSelect = ({ item }) => {
+    if (selected.some(({ value }) => value === item.value)) {
+      setSelected((selected) => selected.filter(({ value }) => value != item.value));
+    } else {
+      setSelected((selected) => [...selected, item]);
+    }
+  };
+
   return (
-    <IconButton
-      accessibilityLabel="Open edit modal to edit Board"
-      icon="check"
-      onClick={() => {setSelected(!selected)}}
-      selected={selected}
-    />
+    <Flex justifyContent="center">
+      <IconButton
+        accessibilityControls="accessibility-example"
+        accessibilityExpanded={open}
+        accessibilityHaspopup
+        accessibilityLabel="More Options"
```

Comment:
This should be "Create Pin Menu" instead

Reply:
And docs above would need to be updated too

Reply:
DONE